docs/architecture/adr-011-transfer-total-escrow-state-entry.md

```diff
@@ -0,0 +1,145 @@
+# ADR 011: ICS-20 state entry for total amount of tokens in escrow
+
+## Changelog
+
+* 2023-05-24: Initial draft
+
+## Status
+
+Accepted and applied in v7.1 of ibc-go
+
+## Context
+
+Every ICS-20 transfer channel has its own escrow bank account. This account is used to lock tokens that are transferred out of a chain that acts as source of the tokens (i.e. when the tokens being transferred are not returning to the originating chain). This design makes it easy to query the balance of the escrow accounts and find out the total amount of tokens in escrow in a particular channel. However, there are use cases where it would be useful to figure out the total amount of a given denomination that is in escrow across all the escrow accounts of the channels where those tokens have been transferred out.
```

Comment:
maybe this reads a little better (slightly less wordy)

```suggestion
Every ICS-20 transfer channel has its own escrow bank account. This account is used to lock tokens that are transferred out of a chain that acts as source of the tokens (i.e. when the tokens being transferred are not returning to the originating chain). This design makes it easy to query the balance of the escrow accounts and find out the total amount of tokens in escrow in a particular channel. However, there are use cases where it would be useful to determine the total escrowed amount of a given denomination across all channels where those tokens have been transferred out.
```